ray-operator/apis/ray/v1/rayservice_types.go

```diff
@@ -20,6 +20,12 @@ const (
 	FailedToUpdateService            ServiceStatus = "FailedToUpdateService"
 )
 
+type UpgradeStrategy string
+
+const (
+	BlueGreenUpgrade UpgradeStrategy = "BlueGreenUpgrade"
```

Comment:
There should also be a second upgrade strategy for "no zero downtime" where we just delete the old cluster and add a new one

Reply:
Oops, yes.